databases/week3/Meal_application.sql

```diff
@@ -0,0 +1,372 @@
+--reservation data and values
+
+insert into reservation (number_of_guests, meal_id, created_date) 
+values (4, 3,'2020-02-11 12:41:00');
+insert into reservation (number_of_guests, meal_id, created_date) 
+values (6, 2,'2020-02-14 20:00:00');
+insert into reservation (number_of_guests, meal_id, created_date) 
+values (2, 6,'2020-02-11 17:00:00');
+
+
+--------------------------------------------
+
+--review data
+insert into review (title, description, meal_id, stars, created_date) 
+values ('Excellent', 'Waaaw', 5, 3,'2020-07-07 00:00:00');
+
+insert into review (title, description, meal_id, stars, created_date) 
+values ('Delicious!!', 'Thanks', 3, 4, '2020-09-03 00:00:00');
+
+insert into review (title, description, meal_id, stars, created_date) 
+values ('Very good', 'will come again soon', 2, 4, '2020-02-12 00:00:00');
+
+
+---------------------------------------------
+
+-- Meal Queries to write
+
+-- Get all meals...
+
+SELECT * FROM Meal;
+
+-- Add a new meal...
+
+INSERT INTO Meal (Title, `Description`, Location, `When`, Max_reservations, Price, Created_date) 
+VALUES ('Tabouleh', 
+'persille', 
+'København H', 
+'2021-01-10 14:30:00', 
+'6',
+29, 
+'2020-01-08 00:00:00'
+); 
+
+-- Get a meal with any id
+
+SELECT * FROM meal 
+WHERE id = 6;
+
+-- Update a meal with any id
+
+UPDATE Meal 
+SET Title = "،Kabab", `Description` = "Yammiii", Location = "Nakskov", `When` = "2020-07-16 13:00:00" , Max_reservations = "15", Price = 40, Created_date = "2020-07-15 18:00:00" 
+WHERE Id = 3;
+
+-- Delete a meal with any id, fx 1
+DELETE FROM Meal WHERE Id = 3;   
+
+
+----------------------------------------------------
+ 
+-- Reservation Queries to write
+
+
+-- Get all reservations
+select * from reservations
+
+
+-- Add a new reservation
+
+INSERT INTO Reservation (Number_of_guests, Meal_id, Created_date) 
+VALUES (
+"2",
+"2", 
+'2020-01-15'
+);
+
+-- Get a reservation with any id...
+
+SELECT * FROM Reservation
+WHERE id = 1;
+
+
+-- Update a reservation with any id...
+
+UPDATE Reservation 
+SET Number_of_guests = 4, Meal_id = 2, Created_date = '2020-01-07'
+WHERE id = 2; 
+
+-- Delete a reservation with any id...
+
+DELETE FROM Reservation 
+WHERE id = 2; 
+
+-------------------------------------------
+
+-- Review Queries to write...
+
+-- Get all Reviews
+
+SELECT * FROM Review;
+
+-- Add a new Review
+INSERT INTO Review (Title, `Description`, Meal_id, Stars, Created_date) 
+VALUES (
+'Excellent', 
+'incredible food',
+'2', 
+4, 
+'2020-02-14'
+);
+
+-- Get a Review with any id...
+
+SELECT * FROM Review
+WHERE id = 1; 
+
+-- Update a Review with any id...
+
+UPDATE Review 
+SET Title = 'Fatoush', `Description` = 'great salad ever' , Meal_id = 4, Stars = 5, Created_date = '2020-02-012'
+WHERE Id = 2; 
+
+-- Delete a Review with any id
+DELETE FROM Review 
+WHERE Id = 3; 
+
+
+-------------------------------------------
+
+-- Additional queries...
+
+-- add a couple of different meals, reservations and reviews with different attributes.
+

```

Comment:
The code repeats itself here.
Please remove the redundant statements.
The tables are already created.